README.md

```diff
@@ -1,6 +1,55 @@
-# curriculum-cr-training-mobile-menu
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# CR TRAINING: MOBILE MENU
 
 
 This repo host an initial code for the tasks performed during the Code Reviewers training.
 
 Application is based on this [Figma design](https://www.figma.com/file/t3EJUCAEViw3QasuJLPLVT/Microverse-Student-Potfolio-Templates-Main?node-id=1%3A1471).
+
+
+## Built With
+
+- HTML
+- CSS
+- JAVASCRIPT
+- VS CODE
+
+
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+
+### Setup
+- Clone the repository
+- cd into the cloned repo directory using the command line
+- Open it in your preferred code editor
+
+## 👤 Author
+
+- GitHub: [@githubhandle](https://github.com/githubhandle)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
```

Comment:
- [ ] **OPTIONAL** I'm sorry this hasn't brought earlier by the previous reviewer, but it's HIGHLY RECOMMENDED that you put your actual valid social links in your README file.
It's important that you let some links to make people (viewers) be able to reach you easily.